thirdparty/fetch-geoip-db.sh

```diff
@@ -13,6 +13,10 @@ filename="GeoLite2-Country.mmdb.gz"
 if [ ! -e $filename ] || [ -n "$(find . -name $filename -mtime +30 -print)" ]; then
 	rm -f $filename
 	echo "Updating GeoIP country database from MaxMind."
-	curl -s -L -O http://geolite.maxmind.com/download/geoip/database/$filename
+        if command -v curl >/dev/null 2>&1; then
+	        curl -s -L -O http://geolite.maxmind.com/download/geoip/database/$filename
```

Comment:
These lines mix tabs with (different numbers of!) spaces.  Please stick to a single tab for each level of indentation.